### ISSUE TITLE
Error in drawing numbers exercise

### ISSUE THREAD

**Issue by RTBagamoyo** [RoboTutorLLC/RoboTutor_2020#130]
I'm have noticed, that the number 9 looks exactly like the letter g. This confused the kids. Furthermore, even though the kids draw the numbers correctly, they still get the exercise wrong.  

Time: Feb 13th, 9am

![image](https://user-images.githubusercontent.com/115608132/221485464-49e17589-fcbc-481c-bef9-1c7213212d18.jpeg)

**Comment by JackMostow** [comment]
You're right that 9 and g look very much alike, but 9 and g are not actually identical in the [Grundschrift font](https://www.cufonfonts.com/font/grundschrift) used in RoboTutor's writing activities.  Fortunately, RoboTutor does not try to distinguish them, because no activity includes them both -- it looks either for digits or for letters.  However, the handwriting recognizer is not perfect, and can confuse 6 with 8.
